tests/js/client/server_parameters/www-authenticate-off.js

```diff
@@ -36,12 +40,20 @@ if (getOptions === true) {
 const jsunity = require('jsunity');
 const request = require('@arangodb/request').request;
 
+let connectWith = function(protocol, user, password) {
+  let endpoint = arango.getEndpoint().replace(/^[a-zA-Z0-9\+]+:/, protocol + ':');
+  arango.reconnect(endpoint, db._name(), user, password);
+};
+
 function testSuite() {
   return {
-    testHeader : function() {
-      let result = request({ url: "/_api/version", method: "get" });
-      assertEqual(200, result.status);
-      assertFalse(result.headers.hasOwnProperty('www-authenticate'));
+    testHeader: function() {
+      protocols.forEach((protocol) => {
```

Comment:
nit: when the test starts, we are connected with HTTP/1, but when we are at the end of this test, we will be connected with HTTP/2. So this test has a side-effect that it permanently changes the connection type.
Ideally we ensure with setUp/tearDown that there are no side-effects after a test completes.